spectrocloud/resource_cluster_group.go

```diff
@@ -282,6 +291,7 @@ func toClusterGroup(c *client.V1Client, d *schema.ResourceData) *models.V1Cluste
 	var values string
 	resourcesObj, ok := d.GetOk("config")
 	endpointType := "Ingress" // default endpoint type is ingress
+	k8Distro := "k3s"
```

Comment:
default is already in the schema. Why it's needed here?

Reply:
changed it